packages/jira-adapter/src/reference_mapping.ts

```diff
@@ -444,20 +444,53 @@ ReferenceContextStrategyName
     serializationStrategy: 'id',
     target: { typeContext: 'parentSelectedFieldType' },
   },
+  {
+    src: { field: 'value', parentTypes: [AUTOMATION_COMPARE_VALUE] },
+    serializationStrategy: 'name',
+    target: { typeContext: 'parentSelectedFieldType' },
+  },
   {
     src: { field: 'values', parentTypes: [AUTOMATION_COMPARE_VALUE] },
     serializationStrategy: 'id',
     target: { typeContext: 'parentSelectedFieldType' },
   },
+  {
+    src: { field: 'values', parentTypes: [AUTOMATION_COMPARE_VALUE] },
+    serializationStrategy: 'name',
+    target: { typeContext: 'parentSelectedFieldType' },
+  },
   {
     src: { field: 'value', parentTypes: [AUTOMATION_FIELD] },
     serializationStrategy: 'id',
     target: { typeContext: 'parentFieldType' },
   },
 ]
 
+// during deploy, serialization method for references
+// from automations is determined by automations values
```

Comment:
1. comment styling nit:
```suggestion
/**
 * during deploy, serialization method for references
 * from automations is determined by automations values
 */
```
2. I might be missing something but I'm not sure how to read this sentence - maybe something like 
> Determine serialization strategy for references with overlapping serialization rules in automation instances: value / values fields will be resolved based on the neighbor type field
3. naming nit: this name is a little too generic, maybe something like `getAutomationValuesLookupFunc`